### ISSUE TITLE
Healing Draught effect is moved to Temporary effect

### ISSUE THREAD

**Issue by GregorHak** [moo-man/WFRP4e-FoundryVTT#2100]
https://github.com/moo-man/WFRP4e-FoundryVTT/issues/1790
Hey, after some tests, I can say that: After You invoke/appy Healing Draught effect then this effect is not proper deleted and stay in Temporary Effect.
![obraz](https://github.com/moo-man/WFRP4e-FoundryVTT/assets/99994770/2cfa81fe-3e7c-43e8-8455-5323101036b8)

To fix that - effect should have this option checked:
![obraz](https://github.com/moo-man/WFRP4e-FoundryVTT/assets/99994770/c2565983-1d02-4432-9cb4-77c8c0fbcb89)

**Comment by Forien** [comment]
I disagree. Instead it should have script to remove it on combat start (or end). 

Healing draughts are limited to 1 use per encounter, so I think it is nice to have some reminder that it was used.

**Comment by GregorHak** [comment]
Ok, or at least a message should be displayed (when you have this effect already) that you can't drink another healing potion. Now these effects are stacked, and that's it.

**Comment by Txus5012** [comment]
Interesting. So maybe something like this?
[Healing-Draught.json](https://github.com/user-attachments/files/15857204/Healing-Draught.json)

Since there is no combat start trigger, I used get initiative to delete the effect.
